### DISCUSSION TITLE
How to turn on local save

### DISCUSSION THREAD

**Discussion by Adri-karry** [Xeno69/Domination#649] [Q&A]
How to enable save in local multiplayer?

**Comment by longtimegamer** [comment]
Autosave should be enabled by default.

Look for this parameter:

https://github.com/Xeno69/Domination/wiki/Params#d_save_to_mpns--save-player-and-mission-progress-to-missionprofilenamespace-if-no-sql-database-is-available-enabling-this-feature-will-also-enable-ranked-gameplay

**Reply by Adri-karry** [reply]
Thanks I will look for the parameter to always make it active to save in local game
